mysql/sql/00_schema_mysql.sql

```diff
@@ -95,29 +96,33 @@ CREATE TABLE `class` (
 --
 
 CREATE TABLE `slot` (
-  `id` INTEGER PRIMARY KEY,
+  `id` INTEGER AUTO_INCREMENT PRIMARY KEY,
```

Comment:
Why did you add the `AUTO_INCREMENT` field?

It should not be there, since the slot ids are inserted via the scrapper